environments/network/ptl.tfvars

```diff
@@ -0,0 +1,107 @@
+enable_debug = "true"
+
+network_address_space                  = "10.10.72.0/21 "
+aks_00_subnet_cidr_blocks              = "10.10.72.0/23"
+aks_01_subnet_cidr_blocks              = "10.10.74.0/23"
+iaas_subnet_cidr_blocks                = "10.10.78.0/23"
+application_gateway_subnet_cidr_blocks = "10.10.70.0/25"
```

Comment:
Not correct to the `network_address_space`

Reply:
all updated